<ui version="4.0" >
 <class>SelectorProperties</class>
 <widget class="QDialog" name="SelectorProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>600</width>
    <height>305</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Selector Control Properties</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="displayPoints_" >
       <property name="text" >
        <string>Display points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="pointProperties_" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Point Properties</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Radius:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="radius_" />
          </item>
          <item>
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Selected:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>16</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>1</number>
             </property>
             <property name="spacing" >
              <number>1</number>
             </property>
             <item row="0" column="0" >
              <widget class="QWidget" native="1" name="selectedColorDisplayer_" >
               <property name="autoFillBackground" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="selectSelectedColor_" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Free:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>16</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>1</number>
             </property>
             <property name="spacing" >
              <number>1</number>
             </property>
             <item row="0" column="0" >
              <widget class="QWidget" native="1" name="freeColorDisplayer_" >
               <property name="autoFillBackground" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="selectFreeColor_" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displayLabels_" >
       <property name="text" >
        <string>Display labels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="labelProperties_" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Point Properties</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Offset:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="labelXOffset_" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="labelYOffset_" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11" >
              <property name="text" >
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="labelSize_" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="labelFont_" />
            </item>
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_3" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>0</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>40</width>
                <height>16</height>
               </size>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" >
               <property name="margin" >
                <number>1</number>
               </property>
               <property name="spacing" >
                <number>1</number>
               </property>
               <item row="0" column="0" >
                <widget class="QWidget" native="1" name="labelColorDisplayer_" >
                 <property name="autoFillBackground" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="labelColor_" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="displayConnectors_" >
       <property name="text" >
        <string>Display connectors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="connectorProperties_" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Connector Properties</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="connectorWidth_" />
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>40</width>
              <height>16</height>
             </size>
            </property>
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>1</number>
             </property>
             <property name="spacing" >
              <number>1</number>
             </property>
             <item row="0" column="0" >
              <widget class="QWidget" native="1" name="connectorColorDisplayer_" >
               <property name="autoFillBackground" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="connectorColor_" >
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>displayPoints_</sender>
   <signal>clicked(bool)</signal>
   <receiver>pointProperties_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>93</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>244</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayLabels_</sender>
   <signal>clicked(bool)</signal>
   <receiver>labelProperties_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>122</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel" >
     <x>465</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayConnectors_</sender>
   <signal>clicked(bool)</signal>
   <receiver>connectorProperties_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>153</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel" >
     <x>247</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
